src/amo/tracking.js

```diff
@@ -239,14 +231,7 @@ export class Tracking {
     }
 
     if (_config.get('server')) {
-      if (dispatch) {
-        const event = { action, category, label, value };
-        dispatch(storeTrackingEvent({ event }));
-      } else {
-        throw new Error(
-          'The dispatch argument must be provided to sendEvent when being called on the server',
-        );
-      }
+      throw new Error('sendEvent: cannot send tracking events on the server');
```

Comment:
Maybe we could add a comment here explaining why the developer received this error